### ISSUE TITLE
feat: add a welcome message dialogue 

### ISSUE THREAD

**Issue by matthiasemde** [matthiasemde/musikus-android#140]
We need a welcome message dialogue to inform users about updates etc. 

I had ChatGPT prepare a message to introduce our open beta: 

>Welcome to Musikus!
>
>Thank you for joining the open beta of. Musikus! We're thrilled to have you on this journey as we shape the future of music practice.
>
>A Work in Progress
>As an open beta, Musikus is still in development, and you may encounter some bugs or incomplete features. Your feedback is invaluable in helping us refine the experience, so please don't hesitate to share your thoughts with us!
>
>A Legacy Continued
>If you're a fan of PracticeTime, you'll feel right at home here. Musikus is the next evolution of that app, built by the same developers, with new tools and features to elevate your practice sessions.
>
>Thank you for being a part of our beta community. Together, we can make Musikus a fantastic resource for musicians everywhere!
>
>Enjoy exploring, and happy practicing!

We could think about having the welcome message query a web address so we can change the message without the user having to update... But that would require Internet permission.

**Comment by mipro98** [comment]
Yeah, the internet think would be very neat, however, I think for now it's not worth adding Internet permissions just for that.
But we might implement an update checker screen or so where we also need internet permissions...

**Comment by matthiasemde** [comment]
@mipro98 we could implement the feature without needing the Internet permission using Firebase remote config. But that would introduce firebase as a dependency 🤷‍♂️
